docs/README.md

```diff
@@ -8,7 +8,7 @@ Redirected to custom domain [https://sdk.up42.com/](https://sdk.up42.com/)
 ## Installation for doc development:
 ```
 cd docs
-pip install -r requirements_dev.txt
+poetry install only docs
```

Comment:
It's confusing that we have poetry install here and make install in dev readme, we need a unified approach